src/extension.ts

```diff
@@ -70,7 +68,7 @@ export function activate(context: ExtensionContext) {
   return Promise.all([
     getCoursierExecutable(context.globalStoragePath),
     parseSmithyBuild(),
-  ]).then(([csBinaryPath, smithyBuild]) => {
+  ]).then(async ([csBinaryPath, smithyBuild]) => {
```

Comment:
**thought:** we could do more than that async-wise, but I wanted a minimal diff for this. If refactors are what we feel would be good to do here, I can also deal with that.